colors/dracula.vim

```diff
@@ -9,203 +9,415 @@
-hi markdownCodeDelimiter ctermfg=228 ctermbg=NONE cterm=NONE guifg=#f1fa8c guibg=NONE gui=NONE
-
-hi markdownBlockquote ctermfg=84 ctermbg=NONE cterm=NONE guifg=#50fa7b guibg=NONE gui=NONE
-hi markdownListMarker ctermfg=84 ctermbg=NONE cterm=NONE guifg=#50fa7b guibg=NONE gui=NONE
-hi markdownOrderedListMarker ctermfg=84 ctermbg=NONE cterm=NONE guifg=#50fa7b guibg=NONE gui=NONE
-hi markdownRule ctermfg=84 ctermbg=NONE cterm=NONE guifg=#50fa7b guibg=NONE gui=NONE
-hi markdownHeadingRule ctermfg=84 ctermbg=NONE cterm=NONE guifg=#50fa7b guibg=NONE gui=NONE
-
-hi markdownUrlDelimiter ctermfg=215 ctermbg=NONE cterm=NONE guifg=#ffb86c guibg=NONE gui=italic"
-hi markdownLinkDelimiter ctermfg=215 ctermbg=NONE cterm=NONE guifg=#ffb86c guibg=NONE gui=italic"
-hi markdownLinkTextDelimiter ctermfg=215 ctermbg=NONE cterm=NONE guifg=#ffb86c guibg=NONE gui=italic"
-
-hi markdownHeadingDelimiter ctermfg=117 ctermbg=NONE cterm=bold guifg=#8be9fd guibg=NONE gui=bold"
-hi markdownUrl ctermfg=141 ctermbg=NONE cterm=NONE guifg=#bd93f9 guibg=NONE gui=NONE
-hi markdownUrlTitleDelimiter ctermfg=84 ctermbg=NONE cterm=NONE guifg=#50fa7b guibg=NONE gui=NONE
-" }}}
-" OCaml {{{
-hi ocamlModule ctermfg=117 ctermbg=NONE cterm=NONE guifg=#8be9fd guibg=NONE gui=italic
-hi ocamlConstructor ctermfg=117 ctermbg=NONE cterm=NONE guifg=#8be9fd guibg=NONE gui=italic
-hi ocamlType ctermfg=141 ctermbg=NONE cterm=NONE guifg=#bd93f9 guibg=NONE gui=NONE
-hi ocamlModPath ctermfg=141 ctermbg=NONE cterm=NONE guifg=#bd93f9 guibg=NONE gui=NONE
-hi ocamlTodo ctermfg=215 ctermbg=NONE cterm=NONE guifg=#ffb86c guibg=NONE gui=italic
-hi ocamlLabel ctermfg=84 ctermbg=NONE cterm=NONE guifg=#50fa7b guibg=NONE gui=NONE
-hi ocamlFullMod ctermfg=117 ctermbg=NONE cterm=NONE guifg=#8be9fd guibg=NONE gui=italic
-hi ocamlWith ctermfg=117 ctermbg=NONE cterm=NONE guifg=#8be9fd guibg=NONE gui=italic
-hi ocamlUnit ctermfg=212 ctermbg=NONE cterm=NONE guifg=#ff79c6 guibg=NONE gui=NONE
-" }}}
+let g:colors_name = 'dracula'
+
+if !(has('termguicolors') && &termguicolors) && !has('gui_running') && &t_Co != 256
+  finish
+endif
+
+" Palette: {{{2
+
+let s:fg        = ['#F8F8F2', 255] | lockvar s:fg
+
+let s:bglighter = ['#424450', 238] | lockvar s:bglighter
+let s:bglight   = ['#343746', 237] | lockvar s:bglight
+let s:bg        = ['#282A36', 236] | lockvar s:bg
+let s:bgdark    = ['#21222C', 235] | lockvar s:bgdark
+let s:bgdarker  = ['#191A21', 234] | lockvar s:bgdarker
+
+let s:subtle    = ['#424450', 238] | lockvar s:subtle
+
+let s:selection = ['#44475A', 239] | lockvar s:selection
+let s:comment   = ['#6272A4',  61] | lockvar s:comment
+let s:cyan      = ['#8BE9FD', 117] | lockvar s:cyan
+let s:green     = ['#50FA7B',  84] | lockvar s:green
+let s:orange    = ['#FFB86C', 215] | lockvar s:orange
+let s:pink      = ['#FF79C6', 212] | lockvar s:pink
+let s:purple    = ['#BD93F9', 141] | lockvar s:purple
+let s:red       = ['#FF5555', 203] | lockvar s:red
+let s:yellow    = ['#F1FA8C', 228] | lockvar s:yellow
```

Comment:
Forgot to mention: The fallback 256-color codes (index 1) of all the colors listed above were chosen by running the actual rgb color of the theme colors through [this library](https://www.npmjs.com/package/x256), which claims to algorithmically choose the closest match.

Not perfect, but close enough for a fallback IMHO.